ui/mirage/scenarios/default.js

```diff
@@ -62,13 +62,14 @@ function jobsIndexTestCluster(server) {
   faker.seed(1);
   server.createList('agent', 1, 'withConsulLink', 'withVaultLink');
   server.createList('node', 1);
+  server.create('node-pool');
```

Comment:
Our mirage environment for testing purposes should show me a node-pool column on job rows.